dtfs-central-api/src/constants/regex.js

```diff
@@ -0,0 +1,5 @@
+exports.UKEF_FACILITY_ID_REGEX = /^\d{8,10}$/;
+
+exports.CURRENCY_NUMBER_REGEX = /^\d+(\.\d{1,2})?$/;
```

Comment:
Tempted to go with no because I want loose validation in central API, but this could be a thing in portal when we first do the validation.
I'm mostly doing data validation from a security perspective in central API rather than business logic validation but happy to chat about this